src/main/resources/data-repository-openapi.yaml

```diff
@@ -634,7 +645,7 @@ definitions:
       createdDate:
         type: string
         description: Date the dataset was created
-      source:
+      contents:
```

Comment:
i think this should still be named source because contents to me is the source with the list of actual objects in the dataset

Reply:
OK, I reverted that change.